src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,82 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 10;
+    private K[] key;
+    private V[] value;
+    private int size;
+
+    public StorageImpl() {
+        value = (V[]) new Object[MAX_ITEMS_NUMBER];
+        key = (K[]) new Object[MAX_ITEMS_NUMBER];
+        size = 0;
+
+    }
```

Comment:
```suggestion
    }
```
no need for this, primitives have default values